presto-main/src/main/java/com/facebook/presto/security/FileBasedSystemAccessControl.java

```diff
@@ -90,8 +89,8 @@ public SystemAccessControl create(Map<String, String> config)
                 ImmutableList.Builder<CatalogAccessControlRule> catalogRulesBuilder = ImmutableList.builder();
                 catalogRulesBuilder.addAll(rules.getCatalogRules());
 
-                ImmutableList.Builder<Pattern> userPatternsBuilder = ImmutableList.builder();
-                userPatternsBuilder.addAll(rules.getUserPatterns());
+                ImmutableList.Builder<PrincipalUserMatchRule> principleUserMatchRulesBuilder = ImmutableList.builder();
```

Comment:
You don't need this builder. Either `ImmutableList.copyOf(rules.getPrincipalUserMatchRules())` or even just pass `rules.getPrincipalUserMatchRules()` directly to `FileBasedSystemAccessControl` and let that constructor do `ImmutableList.copyOf` on its own.